data_steward/gcloud/bq/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+Interact with Google Cloud BigQuery
+"""
+# Python stl imports
+import os
+
+# Third-party imports
+from google.cloud import bigquery
+from google.cloud.bigquery import Client
+from google.auth import default
+from opentelemetry import trace
+from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+# Project imports
+from utils import auth
+
+
+class BigQueryClient(Client):
+    """
+    A client that extends GCBQ functionality
+    See https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.client.Client.html#google.cloud.bigquery.client.Client
+    """
+
+    def __init__(self, project_id: str, scopes=None, credentials=None):
```

Comment:
`project_id` and `credentials` can be accessed anytime because they are attributes of google.cloud.bigquery.client. `scopes` could be made accessible beyond  `__init__(...)` but I haven't found a function/method that would need access to it.